internal/entity/component_instance.go

```diff
@@ -7,10 +7,11 @@ import "time"
 
 type ComponentInstanceFilter struct {
 	Paginated
-	IssueMatchId       []*int64 `json:"issue_match_id"`
-	ServiceId          []*int64 `json:"service_id"`
-	ComponentVersionId []*int64 `json:"component_version_id"`
-	Id                 []*int64 `json:"id"`
+	IssueMatchId       []*int64  `json:"issue_match_id"`
```

Comment:
Do we use IssueMatchId?

Reply:
I think we do from the subresolver:  issuematch -> component_instance